conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureCollections.java

```diff
@@ -71,4 +94,54 @@ public static <T> LinkedHashSet<T> newLinkedHashSet(Iterable<? extends T> iterab
         }
         return set;
     }
+
+    public static <T> List<T> newList() {
+        return new ArrayList<>();
+    }
+
+    public static <T> List<T> newList(Iterable<? extends T> iterable) {
+        Preconditions.checkNotNull(iterable, "iterable cannot be null");
+        if (iterable instanceof Collection) {
+            return new ArrayList<>((Collection<T>) iterable);
+        }
+        List<T> list = new ArrayList<>();
+        for (T item : iterable) {
+            list.add(item);
+        }
+        return list;
```

Comment:
No reason to duplicate code:
```suggestion
        return newArrayList(iterable);
```

Reply:
👍